app/models/message_thread.rb

```diff
@@ -168,10 +172,6 @@ def has_tag_in_message_objects?(tag)
     objects.joins(:tags).where(tags: tag).exists?
   end
 
-  def assign_tag(tag)
```

Comment:
Toto ti presunul linter ci preco to skocilo hore?

Reply:
Doteraz to bola private metoda, to som zmenila